scripts/cibuild.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Consider naming this file `cipublish.sh` instead of `cibuild.sh`. See guidelines [here](https://github.com/azavea/architecture/blob/master/doc/arch/adr-0000-scripts-to-rule-them-all.md#script-summary)